README.md

```diff
@@ -492,6 +495,52 @@ status:
     status: 'True'
 ```
 
+## Custom Binding Paths
+
+There are scenarios where the application resource is not PodSpec-able with custom binding paths for env, volumes, and volumeMounts.  This extension allows a `ServiceBinding` resource to specify custom binding paths for the application resource.  The custom binding paths **MUST** be one or more of env, volumes, and volumeMounts.  In the absence of any of these items, that particular projection **MUST NOT** be performed.  The `ServiceBinding` author **MUST** set the `service.binding/path` annotation to `Custom`.  The binding path value **MUST** be a [JSONPath][jp] location in the application resource.  The binding paths **MUST** be specified in a ConfigMap, and `.spec.application.bindingPaths` in `ServiceBinding` resource **MUST** refer to it.
+
+[jp]: https://kubernetes.io/docs/reference/kubectl/jsonpath/
```

Comment:
JSONPath isn't a standard. kubectl reference docs can provide a notion of what should be possible, but isn't sufficient for an implementor, or to have any degree of conformance.

Reply:
I have changed the approach in the modified text.  But I have avoided the JSONPath.

Reply:
There is a ray of hope: https://mailarchive.ietf.org/arch/msg/jsonpath/igrqSuc_wQ9SHyZBVp76vnhS4hw/